fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -1039,6 +1040,32 @@ public void testIngestOnSubtree() throws Exception {
 
     }
 
+    /**
+     * @author griffinj
+     * Ensure that the objects cannot be pairtree child resources
+     * @see <a href="https://jira.duraspace.org/browse/FCREPO-1505">FCREPO-1505</a>
+     *
+     */
+    @Test
+    public void testIngestOnPairtree() throws Exception {
+        HttpPost method = postObjMethod("");
+        HttpResponse response = client.execute(method);
+        final String location = response.getFirstHeader("Location").getValue();
+
+        // Get the segments from the URL
+        final String[] segments = location.split("/");
+
+        LOGGER.info("POSTing to " + serverAddress + "/" + segments[4] + "/");
```

Comment:
This test seems to be quite brittle, relying on a particular form of URI. Can we switch it to use the actual types on resources?